smriprep/cli/run.py

```diff
@@ -294,38 +278,23 @@ def build_workflow(opts, retval):
     from nipype import logging, config as ncfg
     from ..__about__ import __version__
     from ..workflows.base import init_smriprep_wf
+    from ..utils import Spaces
     from niworkflows.utils.bids import collect_participants
 
-    # Set the default template to 'MNI152NLin2009c'
-    output_spaces = opts.output_spaces or OrderedDict([('MNI152NLin2009cAsym', {})])
+    spaces = Spaces(output=opts.output_spaces)
+    target_spaces = spaces.unique()
+    if not target_spaces:
+        # no output spaces have been set - should this blow up?
```

Comment:
sMRIPrep should not add any space, and should not blow up. 

fMRIPrep will always add MNI152NLin2009cAsym (hence, it shouldn't blow up either)